src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from './types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  selectedOnTab: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId = tabs[0],
+  selectedOnTab,
+}) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId);
```

Comment:
```suggestion
  const selectedTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
```

Reply:
I added first page (tabs[0]) as default value in properties ✔️

So if nothing was selected, we always see the first tab.